Copy to SD Card root directory to update/config.ini

```diff
@@ -173,6 +173,11 @@ files_list_mode:1
 #     TOAST: A non-blocking Toast notification is displayed for few seconds. No user interaction is needed.
 ack_notification:1
 
+#### Notification M117
+#Display of M117 messages on the notification screen
```

Comment:
leave a space after # and put a dot at the end. So "# Display..... screen."

Reply:
Thank you